app/views/general/_opengraph_tags.html.erb

```diff
@@ -9,7 +9,7 @@
 <%= content_for :open_graph_tags %>
 
 <meta property="og:site_name" content="<%= site_name %>" />
-<meta property="og:url" content="<%= request.original_url %>" />
+<meta property="og:url" content="<%= request.original_url.encode("utf-8", invalid: :replace, undef: :replace, replace: "_") %>" />
```

Comment:
Some more info on what caused this would be good. Spec failures I assume?

Would be worth adding any reference material you found that helped diagnose this.

Could we use `force_encoding('utf-8')`?